java/elemental2/chrome/BUILD

```diff
@@ -0,0 +1,39 @@
+# This package contains the build rule to build elemental2-chrome.
+
+package(default_visibility = [
+    "//:__subpackages__",
+])
+
+# Apache2
+licenses(["notice"])
+
+load(
+    "@com_google_jsinterop_generator//:jsinterop_generator.bzl",
+    "jsinterop_generator",
+)
+
+load(
+    "//build_defs/internal_do_not_use:elemental_utils.bzl",
+    "patch_extern_file",
+)
+
+patch_extern_file(
+    name = "chrome_patched",
+    src = "//third_party:chrome.js",
+    patch_file = "chrome.js.diff",
```

Comment:
simple patch is applied via `elemental2`'s patching defs

Reply:
Note I'm working on elemental now in order to get rif of these patch files. This is a burden in term of extern maintenance and require a continuous sync between elemental2 and closure github repo.